README.md

```diff
@@ -82,7 +87,7 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+sentence ::= "the" <adjective>* <noun> <verb> <adverb>?
```

Comment:
That is an EBNF expression. I was looking for something like

~~~
  <sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
  <noun> ::= "dog" | "cat"
  <verb> ::= "ate" | "ran"
  <adverb> ::= "slowly" | "noisily" | ""
~~~

# -5